### PR TITLE
Fixed microad.md

### PR DESCRIPTION
**Pull request by nmochizuki**
Fixed microad.md

### CODE REVIEW

File: ads/microad.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ limitations under the License.
 <amp-ad width="320" height="50"
 	type="microad"
     data-spot="b4bf837c5ddd69758d5ac924d04cf502"
-    data-url=""
-    data-referrer="">
+    data-url="${COMPASS_EXT_URL}"
```
Comment (cramforce) [inline ads/microad.md]: Is this supposed to mean that users of `microad` should insert their respective values?

Comment (nmochizuki) [inline ads/microad.md]: Because these values are automatically generated, users are not required to set them.

Comment (cramforce) [inline ads/microad.md]: K, so people should copy this example literally into their markup?

Comment (nmochizuki) [inline ads/microad.md]: The code above is just an example.  If used in your markup the result will not be specific to your account.  You want to use the MicroAd Compass Interface, and copy the code found inside the box labeled "Ad Tag".

**Comment by cramforce** [comment]
LGTM

**Pull request merged**